inbox/util/testutils.py

```diff
@@ -257,7 +257,7 @@ def folder_status(self, folder_name, data=None):
         lastuid = max(folder_data) if folder_data else 0
         resp = {b"UIDNEXT": lastuid + 1, b"UIDVALIDITY": self.uidvalidity}
         if data and "HIGHESTMODSEQ" in data:
-            resp[b"HIGHESTMODSEQ"] = max(v[b"MODSEQ"] for v in folder_data.values())
+            resp[b"HIGHESTMODSEQ"] = max(v[b"MODSEQ"][0] for v in folder_data.values())
```

Comment:
This area setups fake data for the tests, we don't use this code in real syncs.

`MODSEQ` as stored in each message is always returned as a one element tuple containing an integer e.g. `(123,)` on the protocol level.

The `HIGESTMODSEQ` is a single integer, so it should be 123.

Modification sequences are compared in code as in a > b, means that a is newer than b.

There was an error here since forever that was hidden because of sloppiness of Python 2:

```python
>>> 1 < (1,)
True
```

While on Python 3:

```python
>>> 1 < (1,)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'int' and 'tuple'
``` 

All in all the two tests that rely on this data worked by mistake in Python 2...